.ci/build.sh

```diff
@@ -17,45 +17,57 @@ fi
 
 export DOCKER=docker
 
+if [ "$BUILDSYSTEM" == "autotools" ]; then
+    export LOG_FILES=*/testsuite/libffi.log
+    ./autogen.sh
+else
+    export LOG_FILES=builddir/meson-logs/*
+fi
+
 function build_linux()
 {
-    ./autogen.sh
-    ./configure ${HOST+--host=$HOST} ${CONFIGURE_OPTIONS} || cat */config.log
-    make
-    make dist
-    DEJAGNU=$(pwd)/.ci/site.exp BOARDSDIR=$(pwd)/.ci runtest --version
-    DEJAGNU=$(pwd)/.ci/site.exp BOARDSDIR=$(pwd)/.ci make check RUNTESTFLAGS="-a $RUNTESTFLAGS"
+    if [ "$BUILDSYSTEM" == "autotools" ]; then
+        ./configure ${HOST+--host=$HOST} ${CONFIGURE_OPTIONS} || cat */config.log
+        make
+        make dist
+        DEJAGNU=$(pwd)/.ci/site.exp BOARDSDIR=$(pwd)/.ci runtest --version
+        DEJAGNU=$(pwd)/.ci/site.exp BOARDSDIR=$(pwd)/.ci make check RUNTESTFLAGS="-a $RUNTESTFLAGS"
+    else
+        meson setup builddir
+        meson compile -C builddir
+        meson test -C builddir
+    fi
 
     ./rlgl l --key=${RLGL_KEY} https://rl.gl
-    ./rlgl e -l project=libffi -l sha=${GITHUB_SHA:0:7} -l CC='$CC' ${HOST+-l host=$HOST} --policy=https://github.com/libffi/rlgl-policy.git */testsuite/libffi.log
+    ./rlgl e -l project=libffi -l sha=${GITHUB_SHA:0:7} -l CC='$CC' ${HOST+-l host=$HOST} --policy=https://github.com/libffi/rlgl-policy.git $LOG_FILES
```

Comment:
rlgl is expecting a dejagnu log file.
What is $LOG_FILES?

Reply:
Right, I misunderstood what those commands where doing. I thought it was just about uploading log files, like gitlab does with artifacts. Doing that only for autotools now.

To be honest, I have non idea what dejagnu does. In meson I just run all tests directly and they pass fine (at least on my laptop), but "make check" seems to be doing a lot more work, because it takes super long to complete. I'm probably missing something.